app/models/doi.js

```diff
@@ -91,11 +91,11 @@ const Validations = buildValidations({
     }),
     validator('date', {
       after: '999',
-      before: '2023',
+      before: new Date().getFullYear() + 6,
```

Comment:
Could we define a constant (or computed property or something) for the max year in the future? maybe max_mint_future_offset or something?
Then this would be  max_mint_future_offset + 1

Then below just max_mint_future_offset

Otherwise I can totally see someone changing this in future but forgetting to up this number by 1.

Reply:
@richardhallett, @kjgarza - I just committed changes that make this accessible from an environment variable which supplies the value to a computed field in doi.js:

![image](https://user-images.githubusercontent.com/4681304/180838098-fb85c434-463c-4b92-9179-1b9126b12400.png)

I had to make other changes to get it to work.  If you want any changes, just let me know.

Committing my changes to the 'future-dois' branch did not run the 'pull-request' tests for some reason.

An aside, this branch is way behind 'main' so it must be merged again to bring in everything that has happened since it was last merged.